docs/BMI-Installation.md

```diff
@@ -0,0 +1,288 @@
+# BMI
+
+
+## Installation
+
+This document describes how to setup BMI
+
+### Prerequisite Softwares
+
+BMI requires the following non python dependencies. 
+
+#### Ceph Client
+Ceph is a distributed object store and file system designed for performance and scalibility. 
+For more information, visit the [official website](http://docs.ceph.com/docs/master/start/)
+
+You can follow [this guide](http://palmerville.github.io/2016/04/30/single-node-ceph-install.html) 
+to setup a ceph client on a VM
```

Comment:
It seems the link points to single node ceph installation. What we need is a client.(and it will vary from from one OS to other). So, we should be explicit in mentioning it.(the link points to ubuntu and can work for all debian flavours, I guess).